ComponentKit/TransactionalDataSources/Common/CKTransactionalComponentDataSourceChangeset.mm

```diff
@@ -7,11 +7,14 @@
  *  of patent rights can be found in the PATENTS file in the same directory.
  *
  */
-
+#import "CKTransactionalComponentDataSourceChangeset.h"
 #import "CKTransactionalComponentDataSourceChangesetInternal.h"
 
+#import <UIKit/UITableView.h>
```

Comment:
would prefer we don't depend on UITableView, but just do it manually, but oh well